### PR TITLE
Fix part of issue #156: Additional changes to LaCE Onboarding guide

### PR DESCRIPTION
**Pull request by SanjaySajuJacob**
This PR aims to add images to aid the information provided as well as add additional information.

### CODE REVIEW

**Comment by gp201** [comment]
@SanjaySajuJacob are you still working on this PR?

**Comment by SanjaySajuJacob** [comment]
Yes @gp201 , Praful and I are currently working on the same issue #156. We will have this closed before this Sunday. Thanks!

File: LaCE-onboarding-guide.md [hunk 1]
```diff
@@ -53,6 +71,8 @@ This page will help you solve common queries that you may face while working on
 
 ## 4. Creating a Classroom
 
+![config-tab](https://github.com/SanjaySajuJacob/oppia-web-developer-docs/assets/70763993/2f73ea74-d2ff-4e09-a962-be049bfcb442)
```
Comment (prafulbbandre) [inline LaCE-onboarding-guide.md]: This is not needed as it will be same for rows from 77 to 83

File: LaCE-onboarding-guide.md [hunk 2]
```diff
@@ -12,34 +12,85 @@ This page will help you solve common queries that you may face while working on
 ## 1. Creating a Topic 
 - To create a new topic, we first navigate to the `Topics and Skill Dashboard` and select the `CREATE TOPIC` option. To do so, you can take a look at this [guide](Editor-pages.md#steps-to-create-a-topicskill)
 - A modal will open up with necessary details of the topic that are to be filled such as topic name, topic thumbnail, etc.
+
+![topic-modal](https://github.com/SanjaySajuJacob/oppia-web-developer-docs/assets/70763993/038e469a-d236-4147-bfd8-93919e98f587)
```
Comment (seanlip) [inline LaCE-onboarding-guide.md]: Can you replace the parts in "[...]" with alt text that describes the image, e.g. "Screenshot of the 'New Topic' modal"? Ditto for all other images.

**Review by seanlip — CHANGES_REQUESTED** [review]
Hi @SanjaySajuJacob, looks great in general! Just one comment re a11y. Thanks!

**Review by prafulbbandre — APPROVED** [review]
Looks Good to me.

File: LaCE-onboarding-guide.md (hunk 2 above)
Comment (SanjaySajuJacob) [reply LaCE-onboarding-guide.md]: Hi @seanlip , I have made the changes as you had mentioned here for all the images. Thanks!

File: LaCE-onboarding-guide.md (hunk 1 above)
Comment (SanjaySajuJacob) [reply LaCE-onboarding-guide.md]: Hi @prafulbbandre , I this section is being worked on in your PR as of now. Thanks!

File: LaCE-onboarding-guide.md [hunk 3]
```diff
@@ -12,34 +12,85 @@ This page will help you solve common queries that you may face while working on
...
     - Problem
+    
+    ![Screenshot of problem section](images/LaCEOnboardingGuide/question-problem.png)
+
     - Interaction
-    - Answers ans Responses
+    
+    ![Screenshot of interaction and answers and responses sections](images/LaCEOnboardingGuide/question-interaction-answers-responses.png)
+
+    - Answers ans Responses (This section can be seen in the image above)
```
Comment (seanlip) [inline LaCE-onboarding-guide.md]: ans --> and

**Review by seanlip — APPROVED** [review]
One typo. Please fix, otherwise this is good to merge. Thanks!

File: LaCE-onboarding-guide.md (hunk 3 above)
Comment (SanjaySajuJacob) [reply LaCE-onboarding-guide.md]: Hi @seanlip , I have made the change. Thanks!

**Pull request merged**